plugins/premium/tinymcespellchecker.md

```diff
@@ -141,7 +141,7 @@ This option specifies the spellchecker languages that are available to the user,
 tinymce.init({
   selector: 'textarea',
   plugins: 'tinymcespellchecker',
-  spellchecker_languages: 'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
+  spellchecker_languages: 'US English=en-US,UK English=en-GB,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt-PT,Spanish=es,Swedish=sv'
```

Comment:
Shouldn't the `US English` and `UK English` match the changes you did above?

Reply:
No, the demo is supposed to show setting a different value to the defaults.